xchange-coingi/src/main/java/org/knowm/xchange/coingi/service/CoingiAccountService.java

```diff
@@ -0,0 +1,112 @@
+package org.knowm.xchange.coingi.service;
+
+import org.knowm.xchange.Exchange;
+import org.knowm.xchange.coingi.CoingiAdapters;
+import org.knowm.xchange.coingi.CoingiErrorAdapter;
+import org.knowm.xchange.coingi.dto.CoingiException;
+import org.knowm.xchange.coingi.dto.account.CoingiBalances;
+import org.knowm.xchange.coingi.dto.account.TransactionList;
+import org.knowm.xchange.coingi.dto.request.TransactionHistoryRequest;
+import org.knowm.xchange.coingi.dto.request.WithdrawalRequest;
+import org.knowm.xchange.currency.Currency;
+import org.knowm.xchange.dto.account.AccountInfo;
+import org.knowm.xchange.dto.account.FundingRecord;
+import org.knowm.xchange.exceptions.NotAvailableFromExchangeException;
+import org.knowm.xchange.exceptions.NotYetImplementedForExchangeException;
+import org.knowm.xchange.service.account.AccountService;
+import org.knowm.xchange.service.trade.params.DefaultWithdrawFundsParams;
+import org.knowm.xchange.service.trade.params.TradeHistoryParams;
+import org.knowm.xchange.service.trade.params.WithdrawFundsParams;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.List;
+
+public class CoingiAccountService extends CoingiAccountServiceRaw implements AccountService {
+  public CoingiAccountService(Exchange exchange) {
+    super(exchange);
+  }
+
+  @Override
+  public AccountInfo getAccountInfo() throws IOException {
+    CoingiBalances coingiBalances;
+    try {
+      coingiBalances = getCoingiBalance();
+    } catch (CoingiException e) {
+      throw CoingiErrorAdapter.adapt(e);
+    }
+    
+    return CoingiAdapters.adaptAccountInfo(
+        coingiBalances, exchange.getExchangeSpecification().getUserName());
+  }
```

Comment:
it would be easier to read if would surround the entire method with this type of try / catch like:

```
  @Override
  public AccountInfo getAccountInfo() throws IOException {
    try {
      CoingiBalances coingiBalances = getCoingiBalance();
      return CoingiAdapters.adaptAccountInfo(
        coingiBalances, exchange.getExchangeSpecification().getUserName());
    } catch (CoingiException e) {
      throw CoingiErrorAdapter.adapt(e);
    }
  }
```

please apply this hint to all the methods that have this type of try catch

Reply:
agreed, thanks